### ISSUE TITLE
Feature request to take `MANPAGER` env var over `BAT_PAGER` for `batman`

### ISSUE THREAD

**Issue by jgspratt** [eth-p/bat-extras#10]
(Context: I'm taking you up on your suggestion from https://github.com/eth-p/bat-extras/issues/6 )

It is convenient to be able to `bat` out a file to the screen and then see it for the next command, however, it is convenient to be able to view a man page and then close it and see the history of all commands typed before, so the console would look like this:

```
2020-01-25 22:05:21 at 1593 as root@<hostname> in ~
# man ls

2020-01-25 22:05:26 at 1594 as root@<hostname> in ~
# man man

2020-01-25 22:05:36 at 1595 as root@<hostname> in ~
# man less
```

If `batman` could consider `MANPAGER` (if present) over `BAT_PAGER`, this would allow both (1) traditional `cat`ing of files to the screen for reference in typing of the next command and (2) traditional viewing and closing of man pages.

**Comment by eth-p** [comment]
Sorry for the late reply, I've been pretty busy with classes. I would need to rewrite a couple things (or use a hacky workaround), but this should be pretty easy to do. I'll try and get it done soon-ish.

**Comment by jgspratt** [comment]
No problem. Thanks for the update.

**Comment by eth-p** [comment]
This is now fixed in 5cbbbb6.

I'll close this issue for now. If you encounter any problems, please feel free to reopen it.
